### PR TITLE
Make the continue button public for UI consistency

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
Custom step view controllers do not have access to the rounded button used by ResearchKit to create a consistent look for the continue button. By publicly exposing the rounded button, developers can include a continue button in their custom implementations of `ORKStepViewController` that have the same look as the other steps in a task flow.

### CODE REVIEW

File: ResearchKit.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1069,6 +1069,7 @@
 		FA7A9D321B0843A9005A2BEA /* ORKConsentSignatureFormatter.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = ORKConsentSignatureFormatter.m; sourceTree = "<group>"; };
 		FA7A9D361B09365F005A2BEA /* ORKConsentSectionFormatterTests.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = ORKConsentSectionFormatterTests.m; sourceTree = "<group>"; };
 		FA7A9D381B0969A7005A2BEA /* ORKConsentSignatureFormatterTests.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = ORKConsentSignatureFormatterTests.m; sourceTree = "<group>"; };
+		FB30E8571C7D030F0005AD25 /* ORKTextButton_Internal.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; name = ORKTextButton_Internal.h; path = ../../../ResearchKit/ResearchKit/Common/ORKTextButton_Internal.h; sourceTree = "<group>"; };
```
Comment (YuanZhu-apple) [inline ResearchKit.xcodeproj/project.pbxproj]: Should mark this header file `project`

**Comment by YuanZhu-apple** [comment]
@syoung-smallwisdom Do you need to add a test case in `ORKTest`?

File: ResearchKit.xcodeproj/project.pbxproj (hunk 1 above)
Comment (syoung-smallwisdom) [inline ResearchKit.xcodeproj/project.pbxproj]: Updated

**Comment by syoung-smallwisdom** [comment]
@YuanZhu-apple I'm not sure what kind of a test would be appropriate. This is just exposing a UI element so that custom view controllers can use it.

**Comment by YuanZhu-apple** [comment]
@syoung-smallwisdom It can be as simple as placing the button in a view controller.

**Comment by syoung-smallwisdom** [comment]
@YuanZhu-apple Added test.

**Comment by syoung-smallwisdom** [comment]
@YuanZhu-apple I've moved the Continue Button example into the ORKTest project.

File: Testing/ORKTest/ORKTest/MainViewController.m [hunk 2]
```diff
@@ -371,6 +371,7 @@ - (void)viewDidLoad {
                            @"Wait Task",
                            @"Step Will Disappear",
                            @"Confirmation Form Item"
```
Comment (YuanZhu-apple) [inline Testing/ORKTest/ORKTest/MainViewController.m]: Missed a `,` cause a crash in ORKText

**Comment by syoung-smallwisdom** [comment]
@YuanZhu-apple Fixed the missing comma

File: Testing/ORKTest/ORKTest/ContinueButtonExampleViewController.m [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    
+    [self.footerView.continueButton addTarget:self action:@selector(continueButtonTapped:) forControlEvents:UIControlEventTouchUpInside];
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
```
Comment (YuanZhu-apple) [inline Testing/ORKTest/ORKTest/ContinueButtonExampleViewController.m]: Remove this unused override

File: Testing/ORKTest/ORKTest/ContinueButtonExampleViewController.m [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+/*
+#pragma mark - Navigation
+
+// In a storyboard-based application, you will often want to do a little preparation before navigation
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
+    // Get the new view controller using [segue destinationViewController].
+    // Pass the selected object to the new view controller.
+}
+*/
```
Comment (YuanZhu-apple) [inline Testing/ORKTest/ORKTest/ContinueButtonExampleViewController.m]: And this

File: Testing/ORKTest/ORKTest/ContinueButtonExampleViewController.h [hunk 5]
```diff
@@ -0,0 +1,13 @@
+//
+//  ContinueButtonExampleViewController.h
+//  ORKTest
+//
+//  Created by Shannon Young on 6/27/16.
+//  Copyright © 2016 ResearchKit. All rights reserved.
```
Comment (YuanZhu-apple) [inline Testing/ORKTest/ORKTest/ContinueButtonExampleViewController.h]: Please correct this copyright header

File: Testing/ORKTest/ORKTest/ContinueButtonExampleViewController.m [hunk 6]
```diff
@@ -0,0 +1,45 @@
+//
+//  ContinueButtonExampleViewController.m
+//  ORKTest
+//
+//  Created by Shannon Young on 6/27/16.
+//  Copyright © 2016 ResearchKit. All rights reserved.
```
Comment (YuanZhu-apple) [inline Testing/ORKTest/ORKTest/ContinueButtonExampleViewController.m]: Please correct this copyright header

**Comment by syoung-smallwisdom** [comment]
@YuanZhu-apple Fixed copyright and removed unused code

**Comment by YuanZhu-apple** [comment]
r=me.

**Pull request merged**